buildconfig/class_maker.py

```diff
@@ -234,7 +234,7 @@ class {classname}Test : public CxxTest::TestSuite {{
 }};
 
 
-#endif /* {guard} */""".format(
+#endif /* {guard} */""".format(today=datetime.datetime.now().date().year,
```

Comment:
Could the operation to get the current year be moved to a function given that it is used multiple times.

Reply:
That's a good suggestion, I shall change this now.